content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst

```diff
@@ -25,81 +31,73 @@ depends on the stage of the manufacturing process.
    :guilabel:`Search...` bar to display all virtual locations. From the list, select the
```

Comment:
MENG has recently informed me that we should use the simple "search bar" rather than a GUI label for the search bar, after she saw I was using this very same format. Either way, I think it would be wise to broach this subject with JERO (and/or ZST) to find an agreed-upon solution, because I frequently include the search bar in my docs, and would like to avoid any discrepancies regarding this.

Reply:
Like we talked about this week, and like you confirmed with ZST, I think it would be best to go with a GUI label for the search -- but I *believe* we landed on removing the following "..." from it. If so, I think it would be best to remove them from this line.

Reply:
ZST didn't say anything about removing the "...", just that we can keep the GUI label. I'm going to keep it as is, since that's how it appears in my other docs.

Reply:
Hi there @ksc-odoo @hojo-odoo 

There are two options I think that are acceptable:
- ``` guilabel:`Search...` bar```
- search bar

I prefer the gui in most cases because we are directly referencing it and by mirroring the placeholder text as it is on the UI, we're removing any/all doubt or chances of confusion as to what we're talking about. But it's up to the writer for what they want to do because both are *technically* correct.

> but I believe we landed on removing the following "..." from it.
Don't know where this came from 😄 ``` guilabel:`Search` bar``` isn't correct, to me, because that's not how it's labeled on the UI. If we do the gui version, please include the ellipses.

cc: @jero-odoo